developer/messages/page-is-not-editable.mdx

```diff
@@ -0,0 +1,24 @@
+---
+# this page is referenced in Leo, please do not rename without updating the reference in:
+# https://github.com/makeswift/cosmos/blob/master/packages/leo/src/lib/docs.ts
+
+title: This page is not editable. There are no available elements or page props to edit.
+---
+
+## What this message means
+
+The page you are trying to edit is not editable in Makeswift. This can happen for a number of reasons:
+
+- the page does not support editing in Makeswift
+- your Next.js app is unable to fetch the page metadata due to misconfiguration
+- there was an error fetching the page metadata
```

Comment:
What does this mean? Is this about not being able to fetch the manifest? What's the "page metadata" we're referring to here?

Reply:
I meant connection to the host API. Not sure how we call it.

Reply:
@migueloller  Maybe this?

>The page you are trying to edit is not editable in Makeswift. This can happen for a number of reasons:
>
> - the page is not integrated with Makeswift
> - your Next.js app is unable to communicate with the builder due to misconfiguration
> - there was an error fetching data from the builder

Reply:
Ah, got it. What particular request to the host API? We send a request for the manifest, but that has its own error state. Other requests include fonts and revalidation but those don't block the page from rendering.

It's still not clear to me what the last 2 bullet points mean and if there is something actionable for the user to do here. For example, if I see "unable to communicate with the builder due to misconfiguration" I'll be like "ok, so how do I fix it"? Same with the third bullet point. So I was wondering what you meant by those to see if we could have a more actionable message for the user.